package-lock.json

```diff
@@ -1395,6 +1395,15 @@
         "@bbc/psammead-visually-hidden-text": "^1.2.2"
       }
     },
+    "@bbc/psammead-calendars": {
```

Comment:
Yeah I asked @sareh about this. Because `@bbc/psammead-calendars` is only being used in storybook for this component, all storybook dependencies should be added at the the top level rather than component level.

Reply:
As an example, the [`index.stories.jsx`](https://github.com/bbc/psammead/blob/0eac2370cf1200bcdfdddc70f1a11f2c9b2b58c3/packages/containers/psammead-timestamp-container/src/index.stories.jsx#L14) for timestamp containers has `moment-locales` imported in but it isn't added as a [dependency](https://github.com/bbc/psammead/blob/latest/packages/containers/psammead-timestamp-container/package.json) at component level